test/integration/verify/controls/azurerm_sql_servers.rb

```diff
@@ -0,0 +1,9 @@
+resource_group = attribute('resource_group', default: nil)
+
+control 'azurerm_sql_servers' do
+  only_if { ENV['SQL'] }
+
+  describe azurerm_sql_servers(resource_group) do
+    it { should exist }
```

Comment:
Here would be good to test the plural resource parameters defined above e.g. can check for the `sql_server_name`:
```
its('names'){should include sql_server_name}
```

Reply:
👍